.travis.yml

```diff
@@ -1,14 +1,34 @@
-sudo: false # enable container based builds (can't use sudo)
+sudo: required
+
 language: cpp
-script: make tarball
 
 compiler:
     - gcc
-    # - clang
 
 os:
-    - linux
     - osx
+    - linux
+
+env:
+    global:
+    - DEPS_UBUNTU="g++ cmake libboost-regex-dev xfslibs-dev libssl-dev default-jdk python-dev libfuse-dev libkrb5-dev"
+    - DEPS_CENTOS="gcc-c++ make cmake boost-devel krb5-devel xfsprogs-devel openssl-devel python-devel fuse-devel java-openjdk java-devel libuuid-devel"
+    matrix:		
+    - CONTAINER_LINUX_DISTRO=none
+    - CONTAINER_LINUX_DISTRO=ubuntu VER=14.04
+    - CONTAINER_LINUX_DISTRO=centos VER=6
+
+matrix:
+    exclude:
+    - os: osx
```

Comment:
Oops, you're right. I completely misread this code. These are invalid configurations. Not the configurations we are trying to use. This looks good.